docs/who-uses-meson-python.rst

```diff
@@ -0,0 +1,34 @@
+.. SPDX-FileCopyrightText: 2023 The meson-python developers
+..
+.. SPDX-License-Identifier: MIT
+
+.. _who-uses-meson-python:
+
+**************************
+Who uses ``meson-python``?
+**************************
+
+Here's a curated list of projects using ``meson-python``.
+
+
+.. list-table::
+   :widths: 20 80
```

Comment:
I haven't see how this is rendered, but I don't know if a table is the best choice. Maybe a simple itemized list is better.

Reply:
I tried that, but preferred the list.